packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
   * Migrate docs from terra-layout
   * Added docs for `terra-filter-pills`
   * Added tests for `terra-filter-pills`
+  * Migrate docs from terra-disclosure-manager
```

Comment:
This needs to be moved to the Unreleased section.

Reply:
changed 36eceb6d6a3334407b7ec45cd0baf965d254b662